cluster/gce/gci/configure-helper.sh

```diff
@@ -1268,6 +1268,15 @@ current-context: service-account-context
 EOF
 }
 
+function create-kube-scheduler-config {
+  echo "Creating kube-scheduler config file"
+  mkdir -p /etc/srv/kubernetes/kube-scheduler
+  cat <<EOF >/etc/srv/kubernetes/kube-scheduler/config
+${KUBE_SCHEDULER_CONFIG}
```

Comment:
It would be nice if we also populated this in kube-up. It seems like right now this is only populated in the configuration server.

Reply:
We decided for that to be out of scope https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/785-scheduler-component-config-api#non-goals

But we could set a sample in a follow up. Who are the users of kube-up?

Reply:
The users of kube-up I am aware of include e2e tests in OSS, anyone bringing up Kubernetes directly in GCP and as a reference implementation for bringing up Kubernetes in GCP.

Reply:
Scale testing may also use kube-up.

Reply:
To clarify, you basically mean to add a working sample here https://github.com/kubernetes/kubernetes/blob/868efab6f59632087a47933327e1d7c5b64e3173/cluster/gce/util.sh#L1112, correct?

Reply:
Anywhere under https://github.com/kubernetes/kubernetes/blob/15e95e48964d4e800f7210a249f9122ee7d79fcc/cluster/gce/util.sh#L2700 should work.
I believe your linked in area is under there, so should be fine.